DEVELOPING.md

```diff
@@ -19,37 +21,58 @@ Then to build truss and it's two protoc plugins to your $GOPATH/bin directory:
 $ go install github.com/TuneLab/go-truss/...
 ```
 
+Both can be done from the Makefile in the root directory:
+
+```
+$ cd $GOPATH/github.com/TuneLab/go-truss
+$ make
+```
+
 ## Testing
 
-Before submitting a pull request always run tests that cover modified code. Also build truss and run truss's integration test. This can be done by
+Before submitting a pull request always run tests that cover modified code.
+Also build truss and run truss's integration test. This can be done by
 
-```
-$ cd $GOPATH/src/github.com/TuneLab/go-truss/truss
-$ make clean && make && make test
-# If the tests failed and you want to remove generated code
-$ make testclean
+``` 
```

Comment:
There is a trailing space after this triple grave.

Additionally, this should mention that following these instructions will not run the integration tests, with instructions or directions to the instructions, on how to run the integration tests.